### ISSUE TITLE
Deleting files, but not freeing up space.

### ISSUE THREAD

**Issue by ghost** [diessica/slack-delete-files#16]
Has anyone else experienced this? The files are definitely being deleted, but Slack is not recognizing the free space. If you search for a file that was deleted through the script, it doesn't give you the message "This file was deleted". Instead, it's just a blank space. Check out the attachments.

Any ideas here?

Manual Delete:
![file-delete-manual](https://user-images.githubusercontent.com/896292/47915332-b8e3db00-de5f-11e8-99f0-d87f93d55189.png)

Slack-delete-files
![file-delete-api](https://user-images.githubusercontent.com/896292/47915335-bb463500-de5f-11e8-985b-d9042190444f.png)

**Comment by dedmau5** [comment]
I'm getting this aswell. :( I sent in a ticket to slack support to see what they say...

**Comment by ghost** [comment]
@dedmau5 Yeah I found it actually just takes a few days. Slack docs say it can take up to 5 days.

Here's an easier solution if you're interested: https://www.slackdeletron.com/
